Rakefile

```diff
@@ -1,8 +1,11 @@
 require "yast/rake"
 
-Yast::Tasks.submit_to :sle15sp2
```

Comment:
Should we consider adding a `sle15sp2-qr` target or something similar? Or is it maybe too much? In any case, it is not something that we need to decide now.

Reply:
If it only affects this package, then I guess it is fine to do not add more targets.

Reply:
We need only this single package for the QR media, all other changes should normally go to SP2 MU. Adding `sle15sp2-qr` would need an update for `yast-rake`, you need to ensure it's updated in the Jenkins workers, etc... I think it's easier to just modify the `Rakefile` here.